BUILD_VERSIONS

```diff
@@ -2,5 +2,4 @@
 export SYNAPSE_VERSION=v1.35.1
 export POSTGRES_VERSION=10
 export NGINX_VERSION=1.17
-export RAIDEN_VERSION=v3.0.0rc9
```

Comment:
this isn't needed any more?

Reply:
From the commit message and PR description: 

> Please note that the RAIDEN_VERSION got removed, because it is unused (only references by the also unused purger). As we are currently in a transitioning phase to depract the raiden package, there is no official PyPI release anymore. Thereby it is hard to represent the current state for this version. But as it is not used anyway, it has been decided to remove it completely to avoid any confusion.